### PR TITLE
sort_complex_function

### LINKED ISSUE #12468: sort_complex

### PR DESCRIPTION
**Pull request by ndurumo254**
close #12468

### CODE REVIEW

**Comment by ndurumo254** [comment]
close #12469

**Comment by ndurumo254** [comment]
#12469

File: ivy/functional/frontends/jax/numpy/searching_sorting.py [hunk 1]
```diff
@@ -135,3 +135,9 @@ def sort(
 
 def flatnonzero(a):
     return ivy.nonzero(ivy.reshape(a, (-1,)))
+
+
+@to_ivy_arrays_and_back
+def sort_complex(a):
+    return ivy.sort(a)
+r
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/jax/numpy/searching_sorting.py:143]:
```suggestion
```

**Review by hirwa-nshuti — APPROVED** [review]
LGTM!
Going to merge😊

**Comment by ndurumo254** [comment]
close #12469

**Pull request merged**